FlowCrypt/Functionality/PhotosManager/PhotosManager.swift

```diff
@@ -28,21 +30,19 @@ enum PhotosManagerError: Error {
 
 class PhotosManager: PhotosManagerType {
 
-    enum MediaType {
-        static let image = "public.image"
-    }
-
     func selectPhoto(
         source: UIImagePickerController.SourceType,
```

Comment:
`source` parameter isn't used inside function, can be removed

Reply:
done